bookmarks.html

```diff
@@ -62,29 +62,36 @@ <h1 class="mainHeaderTitle">These are your bookmarks</h1>
         <nav>
           <ul class="navbar" style="list-style: none">
             <li>
-              <a href="index.html"
-                ><img
-                  class="navIcon1"
-                  src="assets/home-5591299_1280.png"
-                  alt="home"
-              /></a>
+              <button aria-label="Close" onclick="{}">
+                <a href="index.html"
+                  ><img
+                    class="navIcon1"
+                    src="assets/home-5591299_1280.png"
+                    alt="home"
+                /></a>
+              </button>
```

Comment:
this is the one exception where we dont need a button 😅 . The anchor Tag is enough. Also, you don't need an onclick attribute. The aria-label can be placed on the anchor tag.